packages/next/types/index.d.ts

```diff
@@ -132,10 +132,16 @@ export type GetServerSidePropsContext<
   locales?: string[]
 }
 
-export type GetServerSidePropsResult<P> = {
-  props?: P
-  unstable_redirect?: Redirect
-}
+export type GetServerSidePropsResult<P> =
```

Comment:
Hey @ijjk, this change broke typescript for us, can you check this issue? https://github.com/vercel/next.js/issues/19271